alvr/dashboard/src/dashboard/components/settings_controls/number.rs

```diff
@@ -12,7 +13,10 @@ fn to_json_value(number: f64, ty: NumberType) -> json::Value {
     match ty {
         NumberType::UnsignedInteger => json::Value::from(number.abs() as u64),
         NumberType::SignedInteger => json::Value::from(number as i64),
-        NumberType::Float => json::Value::Number(Number::from_f64(number).unwrap()),
+        NumberType::Float => {
+            let rounded: f64 = format!("{:.6}", number).parse().unwrap();
+            json::Value::Number(Number::from_f64(rounded).unwrap())
```

Comment:
I know this is a remainder from the whole rounding thing, but the reason decimals are messed up in floats is literally because certain decimals can't be stored as floats, if you round them they'll go back to the correct decimal, but if we turn it into a float again the rounding becomes useless.